gateway-js/src/index.ts

```diff
@@ -800,7 +800,7 @@ export class ApolloGateway implements GraphQLService {
       this.state = {
         phase: 'waiting to poll',
         doneWaiting,
-        pollWaitTimer: setTimeout(() => {
+        pollWaitTimer: global.setTimeout(() => {
```

Comment:
@glasser Is this ok? `ts-jest` can't differentiate between `setTimeout` in the browser vs. in node, and they have different return types.

Reply:
Hmm. This is actually a change to non-test code? Maybe we need somebody who is more of a TS/Jest expert to weigh in.

Reply:
I think the trick is to remove `dom` from the `lib` in tsconfig.test.base.json.

However, that makes an error when trying to build `node_modules/apollo-link-http-common`, as used by `apollo-link-http` as used in `reporting.test.ts`.

That's an ancient version of Apollo Link from AC1 or AC2 or something; you could try upgrading to use `HttpLink` from `@apollo/client` instead. Or frankly just change the single use of this link to use `fetch` like `endToEnd.test.ts` does?